docs/kb/integration/defect-dojo-integration.md

```diff
@@ -0,0 +1,117 @@
+---
+tags:
+  - DefectDojo
+  - Integration
+description: How to connect Semgrep and DefectDojo
+---
+
+# How to connect Semgrep and DefectDojo
+
+## Motivation
```

Comment:
```suggestion
```
Users understand that the opening paragraph of your doc is an introduction without the need for a heading :)